nsupdate/main/templates/main/includes/tabbed_router_configuration.html

```diff
@@ -219,18 +219,17 @@
         </table>
     </div>
     <div class="tab-pane" id="inadyn">
-        <h4>inadyn ({% trans "verified with" %} 1.99.11)</h4>
+        <h4>inadyn ({% trans "verified with" %} 2.1)</h4>
```

Comment:
Yep, up and running on Arch Linux ARM. The only concern here is that maybe while in Arch Linux we have the latest version in other distros they are still stuck at the `1.9`. For instance [ubuntu Xenial](http://packages.ubuntu.com/xenial/inadyn)...

Perhaps we can leave both versions in the same tab?